common/lib/xmodule/xmodule/js/spec/video/video_player_spec.js

```diff
@@ -380,22 +380,21 @@ function(VideoPlayer, HLS) {
             });
 
             describe('when the video is playing', function() {
-                beforeEach(function(done) {
+                beforeEach(function() {
                     state.videoPlayer.play();
+                });
 
+                it('call runTimer in seekTo on player', function(done) {
```

Comment:
@muzaffaryousaf I think yes. Here is why. Whenever `call updatePlayTime on player` was failed, we got the below error

```
Error: Timeout - Async callback was not invoked within timeout specified by jasmine.DEFAULT_TIMEOUT_INTERVAL.
```
This error only happens when an sync task timeout which in our case is `jasmine.waitUntil`. So i don't think we need to `jasmine.waitUntil` in `beforeEach` each test. Thats why i moved it to first test. 

Do you see any issues?